web/src/dialog_widget.ts

```diff
@@ -48,6 +48,7 @@ export type DialogWidgetConfig = {
     html_body: string;
     on_click: (e: unknown) => void;
     html_submit_button?: string;
+    html_cancel_button?: string;
```

Comment:
Might a variable like `html_exit_button` or something like that better label both cancel and close or other such "I'm done" cases?

Reply:
Yes, `html_exit_button` seems better. Updated the PR. Also added a commit to rename the `dialog_cancel_button` class.